{"name": "test_wrong_pwd", "status": "passed", "description": "密码输入6次错误", "attachments": [{"name": "log", "source": "dcf7a5a8-934e-404d-a296-16f0872098ca-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "63b7491d-cc5f-4185-b465-397c70ffd0f5-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "0a4bc6f0-1aba-421e-b616-735538fbc94b-attachment.txt", "type": "text/plain"}], "start": 1631503269557, "stop": 1631503270991, "uuid": "ac320b02-37cd-4e77-b406-bee6b4f3154c", "historyId": "7b9480ebb1961c7c14cb69cc7e65ba50", "testCaseId": "e4c5762512a76c681290f066bcc0e861", "fullName": "test_case.Login_case.TestLogin#test_wrong_pwd", "labels": [{"name": "feature", "value": "登录模块"}, {"name": "severity", "value": "critical"}, {"name": "story", "value": "参数填写不正确，登录失败"}, {"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "Login_case"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-KSOAST3"}, {"name": "thread", "value": "6120-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.Login_case"}]}